Source/SvgElement.cs

```diff
@@ -281,7 +282,7 @@ public virtual SvgDocument OwnerDocument
         /// <summary>
         /// Gets a collection of element attributes.
         /// </summary>
-        protected internal virtual SvgAttributeCollection Attributes
+        public virtual SvgAttributeCollection Attributes

```

Comment:
Why do you need this change ?
Attribute properties are public.

Reply:
Not all attributes are available in this way. Especially when using `use` element.

My use case, based on `SvgUse` code:
https://github.com/wieslawsoltes/Draw2D/blob/3939471702dc7a11a4e9e89455c5c3d2cfcc71ea/svg/Svg.Skia/Elements/Use.cs#L37

Reply:
I don't think so.
Should make `SvgUse` class better.